Mathlib/Topology/Algebra/InfiniteSum/Order.lean

```diff
@@ -81,6 +81,15 @@ theorem tprod_le_tprod_of_inj {g : κ → α} (e : ι → κ) (he : Injective e)
   hasProd_le_inj _ he hs h hf.hasProd hg.hasProd
 #align tsum_le_tsum_of_inj tsum_le_tsum_of_inj
 
+@[to_additive]
+lemma tprod_subtype_le {κ γ : Type*} [OrderedCommGroup γ] [UniformSpace γ] [UniformGroup γ]
```

Comment:
Are there no variables that you can reuse?

Reply:
So annoyingly I couldn't figure out how to do this as `α` above is the closest thing, but its a monoid not a group, and when I try adding the `OrderedCommGroup` instance, things break.